config/samples/e2e/bases/network-peering/network2.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: networking.ironcore.dev/v1alpha1
+kind: Network
+metadata:
+  name: network-sample2
+  namespace: dev-team-onmetal-api
```

Comment:
Please use the same namespace here for the peering as the refs here also indicate that we are pairing within the same `namespace`.